src/Wallabag/CoreBundle/Command/InstallCommand.php

```diff
@@ -135,21 +135,13 @@ protected function setupAdmin(OutputInterface $output)
 
         $em->persist($user);
 
-        $pagerConfig = new UsersConfig();
-        $pagerConfig->setUserId($user->getId());
-        $pagerConfig->setName('pager');
-        $pagerConfig->setValue(10);
+        $config = new Config();
+        $config->setUser($user);
+        $config->setTheme('baggy');
+        $config->setItemsPerPage(10);
```

Comment:
Yes, that's the purpose of `parameters.yml.dist`. I didn't use it atm, but we'll have to in the future.